frappe/integrations/oauth2.py

```diff
@@ -115,7 +115,7 @@ def authorize(**kwargs):
 				resp_html = frappe.render_template(
 					"templates/includes/oauth_confirmation.html", response_html_params
 				)
-				frappe.respond_as_web_page("Confirm Access", resp_html, primary_action=None)
+				frappe.respond_as_web_page(_("Confirm Access"), resp_html, primary_action=None)
```

Comment:
Add in the import or use `frappe._()`, currently this is broken: https://github.com/frappe/frappe/actions/runs/11363881792/job/31608755416?pr=28141

Reply:
```suggestion
				frappe.respond_as_web_page(frappe._("Confirm Access"), resp_html, primary_action=None)
```